lib/cn/kvset.c

```diff
@@ -2882,7 +2883,8 @@ kvset_iter_seek(struct kv_iterator *handle, const void *key, s32 len, bool *eof)
 
     iter->last = SRC_NONE;
 
-    *eof = handle->kvi_eof = iter->pti_meta.eof && iter->wbti_meta.eof;
+    if (eof)
+        *eof = handle->kvi_eof = iter->pti_meta.eof && iter->wbti_meta.eof;
```

Comment:
Which caller doesn't need `eof`?  I thought I'd see that change in this PR.  Hoping you don't say "unit test"... :)